src/Controllers/WebhooksController.php

```diff
@@ -218,17 +219,18 @@ public function deleteExistingWebhook(
         $_headers = array (
             'user-agent'    => BaseController::USER_AGENT,
             'Accept'        => 'application/json',
-            'Authorization' => sprintf('Bearer %1$s', Configuration::$oAuthToken->accessToken)
+            'Authorization' => sprintf('Bearer %1$s', Configuration::$oAuthToken->accessToken),
+            'Content-Type' => 'application/json'
         );
 
         //call on-before Http callback
-        $_httpRequest = new HttpRequest(HttpMethod::DELETE, $_headers, $_queryUrl);
+        $_httpRequest = new HttpRequest(HttpMethod::DELETE, $_headers, $_queryUrl, );
```

Comment:
This comma and space at the end of parameters is some leftover? Could be removed?